NitroxModel/DataStructures/GameLogic/FireData.cs

```diff
@@ -0,0 +1,69 @@
+﻿using System;
+using NitroxModel.DataStructures.Util;
+using ProtoBuf;
+
+namespace NitroxModel.DataStructures.GameLogic
+{
+    [Serializable]
+    [ProtoContract]
+    public class FireData
```

Comment:
Maybe a `CyclopsFireData` class instead of having the cyclops members optional?

Reply:
I can create CyclopsFireData, and have FireData as its base.

Reply:
I changed FireData to CyclopsFireData. I looked all throughout the code and couldn't find a single instance of a fire being spawned outside of the Cyclops.

If they so happen to add fires in the future, or we want to, it would be easy to add a Fires.CreateFire to handle it. I narrowed the scope of the current Fires.CreateFire to make it specific to Cyclops. I should probably rename it to Fires.CreateCyclopsFire.